expo-maps/android/src/main/java/expo/modules/maps/googleMaps/GoogleMapsGeoJsons.kt

```diff
@@ -0,0 +1,155 @@
+package expo.modules.maps.googleMaps
+
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+import com.google.maps.android.data.geojson.*
+import expo.modules.maps.GeoJsonObject
+import expo.modules.maps.interfaces.GeoJsons
+import org.json.JSONObject
+
+class GoogleMapsGeoJsons(private val map: GoogleMap) : GeoJsons {
+  private val layers = mutableListOf<GeoJsonLayer>()
+
+  override fun setGeoJsons(geoJsonObjects: Array<GeoJsonObject>) {
+    deleteGeoJsons()
+    geoJsonObjects.forEach { geoJsonObject ->
+      val geoJsonData = JSONObject(geoJsonObject.geoJsonString)
+      val layer = GeoJsonLayer(map, geoJsonData)
+      setDefaultStyleForLayer(geoJsonObject, layer)
+
+      for (feature in layer.features) {
+        setPolygonFeatureStyle(feature, layer)
+        setPolylineFeatureStyle(feature, layer)
+        setMarkerFeatureStyle(feature, layer)
+      }
+
+      layer.addLayerToMap()
+      layers.add(layer)
+    }
+  }
+
+  private fun deleteGeoJsons() {
+    layers.forEach { it.removeLayerFromMap() }
+    layers.clear()
+  }
+
+  private fun setPolygonFeatureStyle(feature: GeoJsonFeature, layer: GeoJsonLayer) {
+    val polygonStyle = GeoJsonPolygonStyle()
+
+    if (feature.hasProperty("strokeColor")) {
+      polygonStyle.strokeColor = colorStringToARGBInt(feature.getProperty("strokeColor"))
+    } else {
+      polygonStyle.strokeColor = layer.defaultPolygonStyle.strokeColor
+    }
```

Comment:
```suggestion
    polygonStyle.strokeColor = if (feature.hasProperty("strokeColor")) {
      colorStringToARGBInt(feature.getProperty("strokeColor"))
    } else {
      layer.defaultPolygonStyle.strokeColor
    }
```
What do you think? In my opinion that looks better, but it is up to you.

Reply:
It also looks better in my opinion, thank you